osv-scanner.toml

```diff
@@ -112,11 +112,3 @@ name = "stdlib"
 ecosystem = "Go"
 license.override = ["BSD-3-Clause"]
 reason = "Unidentified license, remove once https://github.com/google/deps.dev/issues/86 is resolved"
-
```

Comment:
You can also remove:
* github.com/hashicorp/go-cleanhttp
* github.com/hashicorp/go-version

Reply:
I want to keep it because it's in CNCF allowlist, it maybe included by accident later, WDYT?

Reply:
You can take it from git history if needed but it's your call.